src/commands/error_constants.h

```diff
@@ -43,5 +43,7 @@ inline constexpr const char *errValueIsNotFloat = "value is not a valid float";
 inline constexpr const char *errNoMatchingScript = "NOSCRIPT No matching script. Please use EVAL";
 inline constexpr const char *errUnknownOption = "unknown option";
 inline constexpr const char *errUnknownSubcommandOrWrongArguments = "Unknown subcommand or wrong number of arguments";
+inline constexpr const char *errWrongNumArguments = "ERR wrong number of arguments";
```

Comment:
Should we remove ERR here? ( Seems some str has Err, some are not..)

Reply:
I keep it for convenience, since it should be included in both use sites.

Reply:
We cannot remove the ERR prefix since it may introduce a compatibility issue.

> Seems some str has Err, some are not..

It's an old implementation issue, Redis uses ERR/LOADING/NOPROTO to identify different kinds of errors, but we didn't take care of this in the original implementation. So it's really hard to fix them in short time.